mxd/keycloak.tf

```diff
@@ -94,8 +94,13 @@ resource "kubernetes_config_map" "keycloak_env" {
     KEYCLOAK_ADMIN             = "admin"
     KEYCLOAK_ADMIN_PASSWORD    = "admin"
     # the KC_HOSTNAME must be known in advance, so that Keycloak's token contain valid `iss` claims
-    KC_HOSTNAME       = local.keycloak-ip
+
+    KC_HOSTNAME = local.keycloak-ip
+    #    KC_HOSTNAME_URL   = "http://localhost:8080"
```

Comment:
I would remove this

Reply:
Will do that. That `KC_HOSTNAME_URL` instead of `KC_HOSTNAME` is  needed to work Keycloak UI.

Reply:
Done